subprojects/gradle/module-dependencies-tree/src/main/kotlin/com/avito/module/dependencies/ModuleConfigurationDependencies.kt

```diff
@@ -0,0 +1,16 @@
+package com.avito.module.dependencies
+
+import com.avito.module.configurations.ConfigurationType
+import org.gradle.api.Project
+import org.gradle.api.internal.artifacts.dependencies.DefaultProjectDependency
+
+fun Project.dependenciesOnProjects(configurationType: ConfigurationType): Set<DefaultProjectDependency> {
```

Comment:
the first step for reusing code in impact and new module for demo-apps dependencies